app/main.py

```diff
@@ -1,3 +1,47 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
```

Comment:
```suggestion
    def __init__(self, km: float | int) -> None:
```
The type hint for km should be int | float since integers like Distance(20) are used in the examples.